Makefile

```
@@ -148,21 +148,21 @@ $(GO_APP_BUILD_TARGETS): build-%:
 		go build -ldflags "-s -w -X $(LDFLAGS_VERSION)=$(VERSION) -X $(LDFLAGS_HASH)=$(HASH) -X $(LDFLAGS_BUILDDATE)=$(BUILDDATE)" \
 		-o bin/$* -mod=vendor cmd/$*/$*.go
 
-.PHONY: clean-web
-clean-web:
+.PHONY: clean-console
```

Same for the other rules.

```suggestion
.PHONY: clean-web-console
```